tasks/sign-base64-blob/README.md

```diff
@@ -25,3 +25,7 @@ data:
         pipelineImage: <image pullspec>
         configMapName: <configmap name>
 ```
+## Changelog
+
+## Changes since 1.0.0
```

Comment:
can you make this say `Changes in 1.0.1`?

Reply:
Done!